### DISCUSSION TITLE
如何在bfe_ingress中使用自定义的模块?

### DISCUSSION THREAD

**Discussion by didoodidoo** [bfenetworks/bfe#974] [Q&A (问题和回答)] [answered]
请问如果使用bfe_ingress,那么原生的自定义模块还能支持吗?有没有相关的文档可供参考?

**Comment by mileszhang2016** [comment, marked as answer]
目前在bfe ingress中支持的BFE扩展功能还比较少。
具体可以参考 https://github.com/bfenetworks/ingress-bfe/blob/develop/docs/zh_cn/SUMMARY.md

未来将逐步完善功能。
在2022Q2也会发布支持gateway api标准的ingress。

**Reply by didoodidoo** [reply]
ok,感谢
